js/script.js

```diff
@@ -0,0 +1,82 @@
+/* **************
+ * All Variables *
+ * ************* */
+const bookContainer = document.querySelector('.container--book');
+const bookTitle = document.querySelector('#book-title');
+const bookAuthor = document.querySelector('#book-author');
+const form = document.querySelector('form');
+const storageName = 'libraryStorage';
+let allBooks = [];
+
+// If we don't have a local stroage
+if (!localStorage.getItem(storageName)) {
+  localStorage.setItem(storageName, null);
+}
+
+if (localStorage.getItem(storageName) !== null) {
+  allBooks = JSON.parse(localStorage.getItem(storageName));
+}
+
+// It will add the new book to local storage and allBooks array.
+function addBook(bookTitle, bookAuthor) {
+  if (allBooks === null) {
+    allBooks = [];
+  }
+  allBooks.push([bookTitle, bookAuthor]);
+  localStorage.setItem(storageName, JSON.stringify(allBooks));
+}
```

Comment:
excellent job implementing such functionality but please consider using objects inside the arrays in the local storage instead of using arrays in arrays